astroquery/mpc/tests/test_mpc_remote.py

```diff
@@ -12,17 +12,17 @@ class TestMPC:
     @pytest.mark.parametrize('type, name', [
         ('asteroid', 'ceres'),
         ('asteroid', 'eros'),
-        ('asteroid', 'pallas')])
+        ('asteroid', 'vesta')])
```

Comment:
why this change?

Reply:
The `pallas` query fails for some reason.  Consistently.  No idea why.  I used to work at the MPC, so I could probably ask.  😅 

Anyway, I figured Vesta was a good substitute for Pallas.  And the test passes.

Reply:
I suppose we can just open an issue and ping Mike to ask him to check whether something is off.